Add inline documentation (#90)
nethesis/nethvoice-report: ui/src/doc-inline/en/data_summary_6_table_full.md

# Full Queue Calls

> Data regarding failed calls for *Full Queue* for the selected queues
> in the set period, the data are grouped according to the choice made
> in the filters.

Reviewer: ```suggestion
Data regarding failed calls for *Full Queue* for the selected queues
during the given period.
Data are grouped according to the chosen filter criteria.
```

Failed calls to *Full Queue* are calls that come in
queue found the queue with the maximum number of calls waiting
already achieved and which consequently have not been made to enter failing

> automatically.

Reviewer: "achieved" is wrong in this context, it should be "reached"

The entries shown for calls in *Full Queue* are:

- `Grouping Period`
- `Queue Number`
- `Queue Name`
- `Unique Callers`: total telephone numbers that have entered the queue
counted individually
- `Total Full Queue Calls`

If no queues have been selected, the data of all will be shown

> the queues available.

Reviewer: ?
